Tests/SwiftDocCTests/Model/PropertyListPossibleValuesSectionTests.swift

```diff
@@ -0,0 +1,190 @@
+*/
+
+import XCTest
+import SymbolKit
+import Foundation
+@testable import SwiftDocC
+import SwiftDocCTestUtilities
+
+
+class PropertyListPossibleValuesSectionTests: XCTestCase {
+    
+    func testPossibleValuesDiagnostics() throws {
+        // Check that a problem is emitted when extra possible values are documented.
+        var (url, _, context) = try testBundleAndContext(copying: "DictionaryData") { url in
+            try """
+            #  ``Month``
+            
+            Month object.
+            
+            - PossibleValues:
+              - January: First
+              - February: Second
+              - March: Third
+              - April: Fourth
+            """.write(to: url.appendingPathComponent("Month.md"), atomically: true, encoding: .utf8)
+        }
+        do {
+            XCTAssertEqual(context.problems.count, 1)
+            let possibleValueProblem = try XCTUnwrap(context.problems.first(where: { $0.diagnostic.summary == "\'April\' is not a known possible value for \'Month\'." }))
+            XCTAssertEqual(possibleValueProblem.diagnostic.source, url.appendingPathComponent("Month.md"))
+            XCTAssertEqual(possibleValueProblem.diagnostic.range?.lowerBound.line, 9)
+            XCTAssertEqual(possibleValueProblem.diagnostic.range?.lowerBound.column, 3)
+            XCTAssertEqual(possibleValueProblem.diagnostic.range?.upperBound.line, 9)
+            XCTAssertEqual(possibleValueProblem.diagnostic.range?.upperBound.column, 18)
+            XCTAssertNotNil(possibleValueProblem.possibleSolutions.first(where: { $0.summary == "Remove \'April\' possible value documentation or replace it with a known value." }))
+        }
+        
+        // Check that no problems are emitted if no extra possible values are documented.
+        (url, _, context) = try testBundleAndContext(copying: "DictionaryData") { url in
+            try """
+            #  ``Month``
+            
+            Month object.
+            
+            - PossibleValues:
+              - January: First
+              - February: Second
+              - March: Third
+            """.write(to: url.appendingPathComponent("Month.md"), atomically: true, encoding: .utf8)
+        }
+        do {
+            XCTAssertEqual(context.problems.count, 0)
+        }
+        
+        // Check that a problem is emitted with possible solutions.
+        (url, _, context) = try testBundleAndContext(copying: "DictionaryData") { url in
+            try """
+            #  ``Month``
+            
+            Month object.
+            
+            - PossibleValues:
+              - January: First
+              - February: Second
+              - Marc: Third
+            """.write(to: url.appendingPathComponent("Month.md"), atomically: true, encoding: .utf8)
+        }
+        do {
+            XCTAssertEqual(context.problems.count, 1)
+            XCTAssertEqual(context.problems.count, 1)
```

Comment:
nit: this line is repeated
```suggestion
            XCTAssertEqual(context.problems.count, 1)
```